index.test.js

```diff
@@ -0,0 +1,189 @@
+const skeletorStaticFileCopier = require('./index');
+
+const srcFilepath1 = 'src/skeletor';
+const destFilepath1 = 'dest/skeletor';
+const content1 = ['content1.txt', 'content2.txt'];
+
+const srcFilepath2 = 'src/skeletor2';
+const destFilepath2 = 'dest/skeletor2';
+const content2 = ['content1.txt'];
+
+jest.mock('path');
+
+beforeEach(() => {
+    require('fs-extra')._setMockSrc({
+        [srcFilepath1]: content1,
+        [srcFilepath2]: content2
+    });
+});
+
+test('run() returns a complete status object', () => {
+    const config = {
+        directories: [
+            {
+                src: srcFilepath1,
+                dest: destFilepath2
+            }
+        ]
+    };
+
+    const expectedResponse = {
+        status: 'complete',
+        message: '1 directory processed'
+    };
+
+    expect.assertions(1);
+    return skeletorStaticFileCopier().run(config)
+        .then(response => {
+            expect(response).toEqual(expectedResponse);
+        });
+});
+
+describe('handles troublesome configurations', () => {
+    test('run() handles malformed config object', () => {
+        const config = {};
+
+        const expectedResponse = {
+            status: 'error',
+            error: 'Config directories is not found or not an array.'
+        };
+
+        const response = skeletorStaticFileCopier().run(config);
+        expect(response).toEqual(expectedResponse);
```

Comment:
Since run() returns a promise, you need to put your expect() inside of a then(), like you're doing in your first test.  Otherwise, Jest may pass this test even if it fails.  https://facebook.github.io/jest/docs/en/asynchronous.html#promises